src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -1872,17 +1856,6 @@ private synchronized void markChangedOrUnChanged() {
         frame.setWindowTitle();
     }
 
-    /**
-     * Selects a single entry, and scrolls the table to center it.
-     *
-     * @param pos Current position of entry to select.
-     */
-    public void selectSingleEntry(int pos) {
```

Comment:
Is there a corresponding method that does this? I assume that it was unused, but I guess that there are a number of situations where one would like to use it: click on ParserResult warnings, incorrect encoding on save etc.

On the other hand, it should be quite easy to reimplemented and ideally one would like to call it with an entry I assume.